packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1048,11 +1048,11 @@ abstract class State<T extends StatefulWidget> extends Diagnosticable {
   /// If the parent widget rebuilds and request that this location in the tree
   /// update to display a new widget with the same [runtimeType] and
   /// [Widget.key], the framework will update the [widget] property of this
-  /// [State] object to refer to the new widget and then call the this method
+  /// [State] object to refer to the new widget and then call this method
   /// with the previous widget as an argument.
   ///
-  /// Override this method to respond to changes in the [widget] widget (e.g.,
-  /// to start implicit animations).
+  /// Override this method to respond to changes in the [widget] (e.g., to
```

Comment:
Maybe this would be better phrased more like "`Override this method to respond when the [widget] changes (e.g....`"

Reply:
Done.